### PR TITLE
[Accessiblity] Fix alt text for logo

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Adding alt text to logo. See screenshots

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot
image
image

image
image

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Look at the alt text for the in-test and out-of-test logos

# Checklist

Applicable for all code changes.

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
- [x] My changes look good on IE 11+ and Chrome

### CODE REVIEW

File: frontend/src/text_resources.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ let LOCALIZE = new LocalizedStrings({
       dashboardTabTitle: "Dashboard",
       sampleTest: "Sample eMIB",
       statusTabTitle: "Status",
-      psc: "Public Service Commission",
+      psc: "Public Service Commission of Canada",
```
Comment (caleybrock) [inline frontend/src/text_resources.js]: Do we have a translation for the French?

Comment (caleybrock) [reply frontend/src/text_resources.js]: cc @siobhanozege

Comment (siobhanozege) [reply frontend/src/text_resources.js]: @MichaelCherry update the FR to: "Commission de la fonction publique du Canada"

Comment (MichaelCherry) [reply frontend/src/text_resources.js]: The french already had that value

**Pull request merged**